### PR TITLE
<various>: Ensure consistency between SUMMARY titles and section titles.

### PR DESCRIPTION
**Pull request by flexibeast**
Needed for generating a docs.voidlinux.org-style ToC for the PDF version of the Handbook.

### CODE REVIEW

File: src/config/rc-files.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# rc.conf, rc.local and rc.shutdown
+# rc Files
```
Comment (ericonr) [inline src/config/rc-files.md:1]: I think I prefer the original name. What do you think?

File: src/installation/guides/index.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Advanced Installation Guides
+# Advanced Installation
```
Comment (ericonr) [inline src/installation/guides/index.md:1]: Same here.

File: src/config/rc-files.md (hunk 1 above)
Comment (flexibeast) [reply src/config/rc-files.md:1]: i don't mind it, it's just that it's rather long, which is not ideal for the ToC on the Web site. Happy to change it if you think this isn't an issue.

File: src/installation/guides/index.md (hunk 2 above)
Comment (flexibeast) [reply src/installation/guides/index.md:1]: Changed locally.

File: src/config/rc-files.md (hunk 1 above)
Comment (ericonr) [reply src/config/rc-files.md:1]: Just tested it locally, it has the same width as the title for "Session management", so it looks like it should be fine.

**Pull request merged**